boards/FSM/Src/App/App_Signals.c

```diff
@@ -0,0 +1,52 @@
+#include "App_AcceleratorPedal.h"
+#include "App_Signals.h"
+
+struct AcceleratorPedal *_papps;
+struct AcceleratorPedal *_sapps;
+
+static bool IsPappsAlarmActive(void)
+{
+    return App_AcceleratorPedal_IsEncoderAlarmActive(_papps);
+}
+
+static void PappsAlarmCallback(void *context)
+{
+    struct FsmWorld *         world  = (struct FsmWorld *)context;
+    struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+
+    App_CanTx_SetPeriodicSignal_PAPPS_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
+}
+
+static bool IsSappsAlarmActive(void)
+{
+    return App_AcceleratorPedal_IsEncoderAlarmActive(_papps);
+}
+
+static void SappsAlarmCallback(void *context)
+{
+    struct FsmWorld *         world  = (struct FsmWorld *)context;
+    struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+
+    App_CanTx_SetPeriodicSignal_SAPPS_MAPPED_PEDAL_PERCENTAGE(can_tx, 0.0f);
+}
+
+void App_Signals_Init(struct FsmWorld *world)
+{
+    _papps = App_FsmWorld_GetPapps(world);
+    _sapps = App_FsmWorld_GetSapps(world);
+
+    struct SignalCallback callback;
+    struct Signal *       signal;
+
+    callback.function         = PappsAlarmCallback;
+    callback.high_duration_ms = 10;
+    callback.context          = world;
+    signal = App_SharedSignal_Create(0, IsPappsAlarmActive, callback);
```

Comment:
Somewhat of a nit, but I think it's a bit cleaner to init stuff in one statement, ie:
```
    struct SignalCallback callback  = {
        .function         = PappsAlarmCallback,
        .high_duration_ms = 10,
        .context          = world
    }
    struct Signal* signal = App_SharedSignal_Create(0, IsPappsAlarmActive, callback);
```
similarly for down below.

Reply:
Fine with me, fixed!

Reply:
Turns out the problem lies in `App_AcceleratorPedal.c`

I wrote 
```
return accelerator_pedal->is_encoder_alarm_active;
```
instead of 
```
return return accelerator_pedal->is_encoder_alarm_active();
```

Kind of a silly bug. Turns out I just forgot to write `Test_AcceleratorPedal` so this wasn't caught :sweat_smile: